features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/Comment.jsx

```diff
@@ -327,11 +335,56 @@ class Comment extends React.Component {
                                             allComments={allComments}
                                         />
                                     )} */}
+                                        {comment.replies && comment.replies.list.slice(0).map((reply, index) => (
+                                            <>
+                                                <Box ml={8}>
+                                                    {index !== 0 && <Divider className={classes.divider} />}
+                                                    <Grid container spacing={1} className={classes.root}>
+                                                        <Grid item>
+                                                            <Icon className={classes.commentIcon}>account_circle</Icon>
+                                                        </Grid>
+                                                        <Grid item xs zeroMinWidth>
+                                                            <Typography noWrap className={classes.commentText}>
+                                                                {(reply.commenterInfo && reply.commenterInfo.fullName)
+                                                                    ? reply.commenterInfo.fullName : reply.createdBy}
+                                                            </Typography>
+                                                            <Typography noWrap className={classes.commentText} variant='caption'>
```

Comment:
I think it's better to show the raw datetime, in a [tooltip](https://material-ui.com/components/tooltips/) here